docker-compose.yml

```diff
@@ -62,3 +64,12 @@ swagger:
     - './lib/client:/var/usr/client'
   links:
     - service
+hub:
+  image: selenium/hub
+  ports:
+    - "4444"
+firefox:
```

Comment:
This is more my curiosity than a real question to the PR: Why Firefox? :) Is there any advantages to the Firefox webdriver vs ex Chrome? – I know you use Chrome as your primary browser, so it just stands out to me that you went the other way :smile: